app/scripts/modules/amazon/src/serverGroup/configure/wizard/pages/ServerGroupLoadBalancers.tsx

```diff
@@ -79,167 +86,172 @@ class ServerGroupLoadBalancersImpl extends React.Component<
+                        options={vpcLoadBalancerOptions}
+                        value={values.vpcLoadBalancers}
+                        onChange={this.vpcLoadBalancersChanged}
+                      />
+                    )}
                   </div>
                 </div>
               )}
-            {hasVpcLoadBalancers && (
-              <div>
-                <div className="col-md-4 sm-label-right">
-                  <b>VPC Load Balancers</b>
-                </div>
-                <div className="col-md-8">
-                  {vpcLoadBalancerOptions.length > 0 && (
-                    <TetheredSelect
-                      multi={true}
-                      options={vpcLoadBalancerOptions}
-                      value={values.vpcLoadBalancers}
-                      onChange={this.vpcLoadBalancersChanged}
-                    />
-                  )}
-                </div>
+            </div>
+          )}
+          {!refreshed && (
+            <div className="form-group small" style={{ marginTop: '20px' }}>
+              <div className="col-md-8 col-md-offset-4">
+                {refreshing && (
+                  <p>
+                    <span className="fa fa-sync-alt fa-spin" />
+                    <span> refreshing...</span>
+                  </p>
+                )}
+                {!refreshing && (
+                  <p>
+                    If you are looking for a load balancer or target group from a different application, <br />
+                    <a className="clickable" onClick={this.refreshLoadBalancers}>
+                      click here
+                    </a>{' '}
+                    to load all load balancers.
+                  </p>
+                )}
               </div>
-            )}
-          </div>
-        )}
-        {!refreshed && (
-          <div className="form-group small" style={{ marginTop: '20px' }}>
-            <div className="col-md-8 col-md-offset-4">
-              {refreshing && (
-                <p>
-                  <span className="fa fa-sync-alt fa-spin" />
-                  <span> refreshing...</span>
-                </p>
-              )}
-              {!refreshing && (
-                <p>
-                  If you are looking for a load balancer or target group from a different application, <br />
-                  <a className="clickable" onClick={this.refreshLoadBalancers}>
-                    click here
-                  </a>{' '}
-                  to load all load balancers.
-                </p>
-              )}
             </div>
-          </div>
-        )}
+          )}
+        </>
+      );
+    }
+
+    return (
+      <div className="row">
+        {targetGroupSection}
+        {loadBalancersSection}
       </div>
     );
   }
 }
 
-export const ServerGroupLoadBalancers = wizardPage(ServerGroupLoadBalancersImpl);
+export const ServerGroupLoadBalancers = wizardPage<IServerGroupLoadBalancersProps>(ServerGroupLoadBalancersImpl);
```

Comment:
why is this generic argument necessary?  I ask because I want to redo how formik props are passed through the wizard pages

Reply:
It was necessary to get the specific typings for the specific wizard page, but if you change to use typescript generics allows me to not do this... then all the better!